docs/reference/cli.md

```diff
@@ -13,7 +13,7 @@ description: Command-line interace (CLI) reference materials for IPFS, the Inter
 
 This document is autogenerated from <a href='https://github.com/ipfs/docs/blob/master/scripts/cli.sh'>scripts/cli.sh</a>. For issues and support, check out the <a href='https://github.com/ipfs/docs'>docs repository</a> on GitHub.
 
-_Generated on 2019-08-30 18:40:23, from go-ipfs 0.4.22._
+*Generated on 2020-04-05 23:53:21, from go-ipfs 0.5.0-dev.*
```

Comment:
We should do from 0.4.23 which is the current one.

Reply:
Why? Is this page automatically pushed to the website?

I thought your intention is to prepare the docs for the upcoming 0.5.0 release, that we just have to tag it and release the new documentations with the release.

Reply:
Yes, on merge it publishes the website. I just want to update to the current stable release and once 0.5.0 happens we can update again.